app/router.js

```diff
@@ -45,54 +77,50 @@ export async function router(auth, id, payload, verbose) {
     console.log(
       `${id} Closing issue ${payload.issue.html_url}, reason: ${reason} ${actorRequest}`
     );
-    await closeIssue(
-      await getAuthToken(auth, payload.installation.id),
-      sourceRepo,
-      payload.issue.node_id,
-      reason
-    );
+    await closeIssue(authToken, sourceRepo, payload.issue.node_id, reason);
     return;
   }
 
-  const reopenMatches = reopenMatcher(payload.comment.body);
+  const reopenMatches = commands.reopen.matches;
   if (reopenMatches) {
     console.log(
       `${id} Re-opening issue ${payload.issue.html_url} ${actorRequest}`
     );
-    await reopenIssue(
-      await getAuthToken(auth, payload.installation.id),
-      sourceRepo,
-      payload.issue.node_id
-    );
+    await reopenIssue(authToken, sourceRepo, payload.issue.node_id);
     return;
   }
 
-  const labelMatches = labelMatcher(payload.comment.body);
+  const labelMatches = commands.label.matches;
   if (labelMatches) {
     const labels = extractCommaSeparated(labelMatches[1]);
+    const result = await commands.label.enabled(octokit, config, labels);
 
-    console.log(
-      `${id} Labeling issue ${payload.issue.html_url} with labels ${labels} ${actorRequest}`
-    );
-    await addLabel(
-      await getAuthToken(auth, payload.installation.id),
-      payload.repository.owner.login,
-      sourceRepo,
-      payload.issue.node_id,
-      labels
-    );
-    return;
+    if (result.enabled) {
+      console.log(
+        `${id} Labeling issue ${payload.issue.html_url} with labels ${labels} ${actorRequest}`
+      );
+      await addLabel(
+        authToken,
+        payload.repository.owner.login,
+        sourceRepo,
+        payload.issue.node_id,
+        labels
+      );
+      return;
+    } else {
+      await reportError(authToken, payload.issue.node_id, result.error);
+    }
   }
 
-  const removeLabelMatches = removeLabelMatcher(payload.comment.body);
+  const removeLabelMatches = commands["remove-label"].matches;
```

Comment:
I would suggest sticking to camelCase to avoid syntax like this 😅 

```suggestion
  const removeLabelMatches = commands.removeLabel.matches;
```